demo/index.html

```diff
@@ -54,6 +54,7 @@
                     $('#date4').daterangepicker({
                         startDate: moment().subtract({'h': 1}),
                         endDate: moment(),
+                        timezone: '+02:00',
```

Comment:
This can be misleading if somebody opens the demo in a different timezone from this. If we remove the mandatory UTC as per my other comment this can be
```js
timezone: moment().format('UTCZ')
```